apps/web/pages/_/components.tsx

```diff
@@ -2,7 +2,10 @@ import React, { useState } from 'react';
 import styled from 'styled-components';
```

Comment:
this is the page that displays all icons on the same page for dev use

Reply:
This is so helpful 😍